pkg/scheduler/metrics/metrics.go

```diff
@@ -187,6 +187,15 @@ var (
 			StabilityLevel: metrics.ALPHA,
 		}, []string{"type"})
 
+	PodPriority = metrics.NewHistogram(
+		&metrics.HistogramOpts{
+			Subsystem:      SchedulerSubsystem,
+			Name:           "pod_priority",
+			Help:           "Pod priority number",
+			Buckets:        metrics.ExponentialBuckets(1, 2, 15),
```

Comment:
this would create buckets [1, 2), [2, 4), [4, 8)..., [16k, inf), right? In practice, I've seen pod priorities to be usually in the 1000s, which wouldn't give this much granularity (e.g. priority 5000 and 6000 cannot be differentiated). Also, would this capture pod priority 0 (default value)?

Reply:
Yes, I think in this case defining buckets manually is probably most reasonable. Exponential buckets are really meant for buckets of request latencies of a network service.